dbt/include/trino/macros/adapters.sql

```diff
@@ -136,7 +136,7 @@
 {%- endmacro -%}
 
 {% macro comment(comment) %}
-  {%- set persist_docs = model['config'].get('persist_docs') -%}
```

Comment:
I don't mind the implementation, but that's the idea. `model['config']` can return null.

The code should be updated to make it safer.